src/App.js

```diff
@@ -1,20 +1,62 @@
-import React from 'react';
+import React, { createContext, useEffect, useState } from 'react';
 import { hot } from 'react-hot-loader';
 import HeaderComponent from './header/header.component';
 import GlobalStyles from './global/globalStyles';
 import FooterComponent from './footer/footer.component';
 import PostersBlock from './postersBlock/postersBlock.component';
 import ErrorBoundary from './global/components/errorComponent/errorBoundary.component';
+import { getPosters, getPostersByCategory } from './servise/posterService';
+import { BigLine } from './global/components/bigLine/bigLine.styled';
 
-const App = () => (
-  <>
-    <ErrorBoundary>
-      <GlobalStyles />
-      <HeaderComponent />
-      <PostersBlock />
-      <FooterComponent />
-    </ErrorBoundary>
-  </>
-);
+const links = [
+  { id: 0, text: 'all' },
+  { id: 1, text: 'documentary' },
+  { id: 2, text: 'comedy' },
+  { id: 3, text: 'horror' },
+  { id: 4, text: 'crime' },
+];
+
+export const AppContext = createContext(null);
+const App = () => {
+  const [posters, setPosters] = useState([]);
+  const [countPosters, setCountPosters] = useState(0);
+  const [selectedPoster, setSelectedPoster] = useState(false);
+  const [submitForm, setSubmitForm] = useState({ form: null });
+  const [activeNav, setActiveNav] = useState(() => links[0]);
+
+  useEffect(async () => {
```

Comment:
using async as useEffect callback is anti-pattern
https://fernandoabolafio.medium.com/how-to-go-async-with-useeffect-ff74dff0eec6